Formula/gpsim.rb

```diff
@@ -15,10 +14,16 @@ class Gpsim < Formula
 
   depends_on "gputils" => :build
   depends_on "pkg-config" => :build
+  depends_on "gettext"
   depends_on "glib"
   depends_on "popt"
+  depends_on "readline"
 
   def install
+    ENV.cxx11
+
+    inreplace "src/modules.cc", "error.h", "errors.h"
```

Comment:
Is upstream aware of this issue?

Reply:
Emailed the mailing `gpsim-devel` mailing list and also opened a bug ticket on SourceForge: https://sourceforge.net/p/gpsim/bugs/245/.